package.json

```diff
@@ -1,70 +1,19 @@
 {
   "name": "@contentful/create-contentful-app",
-  "version": "0.4.2",
```

Comment:
This should have `"private": true` to avoid accidental publishing. Also, `version` is required (I think).

Reply:
Version is taken from outside (`lerna.json`) and the private switch does not really matter, as publishing is now done by ci

Reply:
But if you run `npm publish`, the regular `npm` publishing still gets triggered, right?